### DISCUSSION TITLE
hide a "relay" from the WEB  UI??

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#18567] [Q&A]
Is there a way to hide a relay defined in a template from the WEB UI.
My understanding is:
If you define a relay in the template it always show up in the webUI and ist switchable!!

I run in some cases into the problem that I need a "virtual" relay for any reason ... but it shouldn't show UP in the UI and also is not switchabe from the UI ...
same for a dimmer  (PWM) .. esp. when u have a LED on the device do dim!!

Thanks
Mark_Muc

**Comment by sfromis** [comment]
Tasmota does indeed not facilitate customization of how you want the regular web UI to be structured, as this is only meant for admin/test. Daily operation is expected to be handled in some smarthome "dashboard" in a backend setup, providing much fuller control than the rudimentary web UI only having a small part of Tasmota features available.

**Reply by Mark-Muc** [reply]
ok I can understand that .. just asking for it!!
MayBe with Berry a UI-Dev Set will be avail so enybody can design a  dashbordlike UI and display what he want.
There is a thread with so Berry scripts but not much "action"
Thx for answering!!

Mark_Muc

**Reply by sfromis** [reply]
With ESP32 and Berry, you can already create an entire extra page in the web UI including whatever elements you want.
